### PR TITLE
sort check list in layer and service page and set check date in service page to US/Estern zone

### PR DESCRIPTION
**Pull request by zhangmingda**
issue#281 and issue#282
The check date stored in database is UTC. If TIME_ZONE sets 'UTC', the pages will show UTC time.

### CODE REVIEW

**Comment by capooti** [comment]
@zhangmingda let's use the default attribute order for this, no need to create a specific method.

You can get this working using the ordering option in meta model options:
https://docs.djangoproject.com/en/1.11/ref/models/options/#ordering

Note: the build is failing. I don't think this is because of your commit. @tomkralidis any idea?

File: hypermap/settings.py [hunk 1]
```diff
@@ -96,7 +96,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'US/Eastern'
```
Comment (capooti) [inline hypermap/settings.py]: great catch. Could you commit this in a specific PR @zhangmingda ?

Comment (zhangmingda) [inline hypermap/settings.py]: Sure, I will commit this separately.

**Comment by zhangmingda** [comment]
Thanks for the document link @capooti

**Comment by zhangmingda** [comment]
Committed another PR.

**Pull request closed without merging**